### ISSUE TITLE
Editing asset (rotation) changes the uploader

### ISSUE THREAD

**Issue by leevigraham** [craftcms/cms#5990]
### Description

Editing an asset changes the uploader. This breaks permissions when an admin edits a users image. Editing also seems to move the file to another folder breaking dynamic folder paths.

### Steps to reproduce

1. Upload an image as a user
2. Edit the file as another user

### Additional info

- Craft version: 3.5 beta1
- PHP version: PHP 7.4
- Database driver & version: Maria DB

**Comment by leevigraham** [comment]
Additionally there's some weirdness when an asset is rotated… The preview changes but editing the file again shows the original file orientation.

![2020-04-29 22 01 24](https://user-images.githubusercontent.com/25124/80593855-414b3800-8a65-11ea-9bc8-dcc2bf42d23e.gif)

**Comment by leevigraham** [comment]
> Editing an asset changes the uploader.

After reading the docs… this is the expected behaviour. It still breaks some expected permissions tho.

**Comment by andris-sevcenko** [comment]
@leevigraham I'm unable to reproduce the rotation issue - might be some caching issue?

**Comment by andris-sevcenko** [comment]
@leevigraham what do you mean with "It still breaks some expected permissions tho."?

**Comment by leevigraham** [comment]
@andris-sevcenko Looking into this a bit more it could be more complicated. I uploaded a bunch of old photos from a website which had oldschool rotation meta issues (iphone 5 days).

I was importing the images directly using a command and getting their height and width with:  Image::imageSize()

<img width="659" alt="image" src="https://user-images.githubusercontent.com/25124/81498352-c0632a80-9307-11ea-92ab-8f9e0c29ea9b.png">

You can see some of the images here: https://mytiki.life/tiki-mugs/unidentified-mug-10266

Chrome is rotating them… but the import has the height and width swapped… hence the wacky ui.

> what do you mean with "It still breaks some expected permissions tho."?

Ignore that for now. I'll try and articulate that better in the future.

**Comment by andris-sevcenko** [comment]
1) By setting scenario to `essentials`, you're bypassing image cleansing routine, which also rotates the image according to EXIF data and then removes the rotation EXIF data. So you're getting an image that is sideways, but has EXIF data, that tells the browser to rotate it the right way. This borked up transforms and other stuff real good, which is why we're removing it from the image. You can call the routine yourself by calling `Craft::$app->getImages()->cleanImage($path)`

2) You really should not be setting custom field values directly as of 3.4 (https://craftcms.com/knowledge-base/updating-plugins-for-craft-34#setting-custom-field-values).

So, to sum up - if you run the image through the `cleanImage` routine, it all should be good.
